RELEASING.md

```diff
@@ -7,6 +7,54 @@
 
 ## Release Procedure
 
+- Do platform test via tox:
+  ```
+  tox -r
+  ```
+
+- FIXME: Make sure docs can be built.
```

Comment:
@LinguList how do you do this?

Reply:
I have instructions here: https://github.com/lingpy/doc/ (in fact, I completely forgot that I wrote this down).

Reply:
So all docs now go to `lingpy/doc/source`, from where they can be turned into the html output with sphinx.